### PR TITLE
Rename file and buffer to filename and input_buffer

### PR DESCRIPTION
**Pull request by jianajavier**

### CODE REVIEW

File: autopsy_dashcam.py [hunk 1]
```diff
@@ -52,29 +52,29 @@ def startUp(self, context):
         self.platform = ''
         pass
 
-    def process(self, file):
+    def process(self, filename):
```
Comment (bartbroere) [inline autopsy_dashcam.py:55]: It's not strictly a ``filename`` (it has some more attributes), maybe ``source_file`` would be a nice variable. It's better than shadowing ``file`` in any case. Thanks for the contribution!

**Review by bartbroere — APPROVED** [review]

File: autopsy_dashcam.py (hunk 1 above)
Comment (jianajavier) [reply autopsy_dashcam.py:55]: Sorry about that! I can change quickly if you'd like, let me know @bartbroere

Comment (bartbroere) [reply autopsy_dashcam.py:55]: Yes, if it's not too much effort. Thanks :sweat_smile:

**Review by bartbroere — APPROVED** [review]

File: autopsy_dashcam.py (hunk 1 above)
Comment (jianajavier) [reply autopsy_dashcam.py]: @bartbroere Updated!

**Pull request merged**